src/index/formatFileStructure/removeEmptyFolders.ts

```diff
@@ -1,18 +1,28 @@
 import fs from "fs";
 import path from "path";
 
-export const removeEmptyFolders = (p: string) => {
-  const files = fs.readdirSync(p);
-  if (!files) {
-    fs.rmdirSync(p);
-  }
-  for (const file of files) {
-    const fullPath = path.join(p, file);
-    if (fs.lstatSync(fullPath).isDirectory()) {
-      removeEmptyFolders(fullPath);
-      if (!fs.readdirSync(fullPath).length) {
-        fs.rmdirSync(fullPath);
-      }
-    }
+/**
+ * @summary
+ * Recursively removes all empty folders.
+ *
+ * @param directory the full path of a directory.
+ *
+ * @example
+ * removeEmptyFolders('/home/user/documents/project/src')
+ */
```

Comment:
I think this is unnecessary due to the self explanatory function name. If anything, we could add a simple comment that explains the intent and recursive nature.

Reply:
I think keeping the summary is ok, because the recursive nature matters.

I'll remove the params.

I think the example is good because my experience working with paths, directories and filenames can be confusing. This clears things up that a resolved path is required, without having to say it's a solved path.

Reply:
I still think `/** Recursively removes all empty folders. */` is more than enough here.

Reply:
Ok, no worries. I'll change it now.